app/obolapi/api_internal_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright © 2022-2024 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package obolapi
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestWithTimeout(t *testing.T) {
+	// no timeout = 10s timeout
+	oapi, err := New("http://url.com")
+	require.NoError(t, err)
+	require.Equal(t, 10*time.Second, oapi.reqTimeout)
```

Comment:
You should have access to `defaultTimeout` const to use it instead of the explicit literal.